src/assets/styles/mixins.less

```diff
@@ -0,0 +1,15 @@
+﻿.right-alignment(@icon-order) {
+  right: (15 + (@icon-order * 30));
+}
+
+.svg-position {
```

Comment:
Would `position-svg` be better (that is what it does)? It specifies position and top for any svg (could be button but also includes the logo) in the toolbar.